docs/validator/validator-faq.md

```diff
@@ -14,32 +14,55 @@ Fisherman (or validator) - nodes that provide security by verifying the validity
 
 ## How do I become a validator?
 
-If you already have an account with sufficient amount of funds.
+You need an account with sufficient amount of funds in the wallet.
 You can follow docs [here](local-setup/running-testnet.md) how to become validators
 
 Specific steps:
-Create new key pair that will be used for staking for given account
-Start a node with this key pair in validator_key.json
+Create new key pair that will be used for staking for given account, and load it with the funds you want to put at stake
+Start a node with the new key pair stored in validator_key.json
 Send a staking transaction using your wallet / CLI with your account including amount and public key from newly generated key pair.
 Wait until node becomes a validator
 
 ## What is 'staking'?
 
-We call staking a process of sending StakeTransaction that inform the network that given account wants to become a validator in upcoming epochs. This transaction must provide a public key and staking amount.
-After this, a node that has a private key associated with the public key in the staking transaction must wait until epoch change to become validator if the staking amount is above the seat price.
+We call staking a process of sending StakeTransaction that inform the network that given account wants to become a validator in upcoming epochs. This particular type of transaction must provide a public key and staking amount.
+After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until the next epoch to become validator. **important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
+
+## What is a minimum amount to stake as a validator?
+
+Near, Inc. has staked 20 NEAR, so staking 10 NEAR should be enough to become a validator on the TestNet. On the MainNet, the minimum amount is dynamic, and is defined by the amount of NEAR tokens put at stake by other validators.
```

Comment:
This will not be true for testnet. Also I am not sure why we mention "NEAR INC" (btw there is no comma between NEAR and INC as well as no period at the end)

Reply:
Removed that part entirely. It was already in the documentation, probably a fossil from previous testnet ages